corehq/apps/translations/app_translations/upload_app.py

```diff
@@ -271,9 +272,11 @@ def _process_rows(app, sheet_name, rows, names_map, lang=None):
     if not sheet_name or not rows:
         return []
 
+    older_sheet_details = get_bulk_app_sheets_by_name(app, lang, eligible_for_transifex_only=True)
```

Comment:
`eligible_for_transifex_only` should be set to False no?

Reply:
how about we use `current` for the current translations instead of `older`/`previous` here?